addons/project/static/tests/project_task_kanban_renderer_tests.js

```diff
@@ -0,0 +1,148 @@
+/** @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { click, getFixture, nextTick } from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+import { start } from "@mail/../tests/helpers/test_utils";
+import { registry } from "@web/core/registry";
+import { makeFakeUserService } from "@web/../tests/helpers/mock_services";
+
+let target;
+let serverData;
+
+const serviceRegistry = registry.category("services");
+
+function getColumn(target, groupIndex = 0, ignoreFolded = false) {
+    let selector = ".o_kanban_group";
+    if (ignoreFolded) {
+        selector += ":not(.o_column_folded)";
+    }
+    return target.querySelectorAll(selector)[groupIndex];
+}
```

Comment:
It is only used 2 times, I dont think using a method is usefull, nor does it make the code clearer.

You can simply do:
```js
 target.querySelectorAll(selector)[groupIndex];
 ```
 at L24 and L139
if L139 is too long, you can create an intermediate variable instead